index.html

```diff
@@ -0,0 +1,532 @@
+            and have a project you need developed,<br> please 
+            do not hestiate to contact me.</p>
+
+        <div class="social-media">
+            <!-- github -->
+            <div class="github frame">
+                <div class="github-icon">
+                    <a href="https://github.com/francis450"><img src="img/social-media-icons/github.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- linkedin -->
+            <div class="linkedin frame">
+                <div class="linkedin-icon">
+                    <a href="https://linkedin.com/in/njoroge-francis"><img src="img/social-media-icons/linkedin.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- peace -->
+            <div class="peace frame">
+                <div class="peace-icon">
+                    <a href="#"><img src="img/social-media-icons/peace.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- twitter -->
+            <div class="twitter frame">
+                <div class="twitter-icon">
+                    <a href="https://twitter.com/nkamandefrancis"><img src="img/social-media-icons/twitter.svg" alt=""></a>
+                </div>
+            </div>
+
+            <!-- m -->
+            <div class="m frame">
+                <div class="m-icon">
+                    <a href="https://medium.com/@franciskamande2001"><img src="img/social-media-icons/m.svg" alt=""></a>
+                </div>
+            </div>
+        </div>
+
+        <div class="background-image">
+            <img src="img/headline-background.svg" alt="">
+        </div>
+    </section>
+
+    <!-- Works -->
+    <section id="recent-works">
+        <div id="title">
+            <h2 id="title-text">My Recent Works</h2>
+            <div id="title-vector"></div>
+        </div>
+
+        <div id="works-frame44">
+            <div id="works-featured-image">
+                <div class="works-image-placeholder">
+                    <img src="img/works/Img-placeholder.svg" alt="">
+                </div>
+                <div id="works-right-block">
+                    <h3 id="works-title-post">Multi-Post Stories</h3>
+                    
+                    <p id="works-supporting-text">
+                        A daily selection of privately 
+                        personalized<br> reads; no accounts 
+                        or sign-ups required.<br> has been 
+                        the industry's standard dummy<br> 
+                        text ever since the 1500s, 
+                        when an<br> unknown printer took 
+                        a standard dummy<br> text.
+                    </p>
+
+                    <div id="works-tags">
+                        <p class="tag">css</p>
+                        <p class="tag">html</p>
+                        <p class="tag">bootstrap</p>
+                        <p class="tag">Ruby </p>
+                    </div>
```

Comment:
- [ ] Kindly ensure that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li` which is semantic as `<ul>` is used to represent a list 👍🏽.